share/shwap/p2p/shrex/shrex_getter/shrex.go

```diff
@@ -247,6 +259,12 @@ func (sg *Getter) GetNamespaceData(
 		return shwap.NamespaceData{}, nil
 	}
 
+	request, err := shwap.NewNamespaceDataID(header.Height(), namespace)
+	if err != nil {
+		return nil, err
+	}
+	response := shwap.NamespaceData{}
```

Comment:
better be pointer? &shwap.NamespaceData{}

Reply:
its a type alias of `[]RowNamespaceData`